inscriptions/ISic1057.xml

```diff
@@ -178,39 +183,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140537">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    ἐνθάδε κῖτε πε͂ς <surplus>²⁶παῖς</surplus>²⁶<lb n="2"/>ὀνόματι Κλωδια-<lb n="3"/>νός· τελευτᾷ μη-<lb n="4"/>νὶ ΚΙ <surplus>²⁷k(a)ḷ(endis)?</surplus>²⁷ Φεβλαρίες <surplus>²⁶Φεβρουαρίαις</surplus>²⁶.
-                    <lb n="5"/> μή τις ἀνύξῃ.
-                    <lb n="6"/>
-                </ab>
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI175475">
-                <ab>
-                    <lb n="1"/>
-
-
-                    ἐνθάδε κῖτε πε͂ς <surplus>²⁶κεῖται παῖς</surplus>²⁶<lb n="2"/>ὀνόματι Κλωδια-<lb n="3"/>νός. τελεύτα
-                    μη-<lb n="4"/>
-                    <expan>
-                        <abbr>ν</abbr>
-                        <ex>ῶν</ex>
-                    </expan>
-                    ιʹ
-                    <expan>
-                        <abbr>κ</abbr>
-                        <ex>α</ex>
-                        <abbr>λ</abbr>
-                    </expan>
-                    (άνδαις) Φεβλαρίες <surplus>²⁶Φεβρουαρίαις</surplus>²⁶.<lb n="5"/> μή τις ἀνύξῃ.
-                    <lb n="6"/>
+                    <lb n="1"/>ἐντάδε κῖτε πε&#x0342;ς
+                    <lb n="2"/>ὀνόματι Κλωδια
+                    <lb n="3" break="no"/>νός · τελευτᾷ <expan><abbr>μη
+                    <lb n="4" break="no"/>ν</abbr><ex>ῶν</ex></expan> <hi rend="supraline"><num value="10">ι</num></hi> <expan><abbr>κ</abbr><ex>α</ex><abbr>λ</abbr><ex>άνδαις</ex></expan> Φεβλαρίες
```

Comment:
There is no supraline above the numeral on this text. I note the abbreviation / ligature for Kalandais is actually much more complex, but that can be left for future editing!